doc/7/protocols/websocket/introduction/index.md

```diff
@@ -13,3 +13,11 @@ Inherits from: [KuzzleEventEmitter](/sdk/js/7/core-classes/kuzzle-event-emitter)
 The WebSocket protocol can be used by an instance of the SDK to communicate with your Kuzzle server.
 
 This protocol allows you to use all the features of Kuzzle, including [real-time notifications](/sdk/js/7/essentials/realtime-notifications).
+
+<SinceBadge version="2.10.0"/>
```

Comment:
We are not sure about the target release version (even if in that case we know pretty much I admit)
```suggestion
<SinceBadge version="change-me"/>
```